### DISCUSSION TITLE
Welcome to comments Discussions!

### DISCUSSION THREAD

**Discussion by SamirPaulb** [SamirPaul1/comments#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samirti** [comment]
![rainbow-superthin](https://github.com/SamirPaulb/comments/assets/134064358/928ac797-b673-48e9-b20f-af4ec89df99d)
